softlayer/vm/fs_agent_env_service.go

```diff
@@ -2,11 +2,17 @@ package vm
 
 import (
 	"encoding/json"
+	"time"
 
 	bosherr "github.com/cloudfoundry/bosh-utils/errors"
 	boshlog "github.com/cloudfoundry/bosh-utils/logger"
 )
 
+const (
+	maxAttempts = 30
```

Comment:
Also, separate the PRs. Remove the retry change from @jianqiu from this PR. I asked him to do the same on his w.r.t. To this tag change. Thx